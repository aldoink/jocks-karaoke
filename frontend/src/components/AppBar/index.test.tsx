import 'jest-styled-components';
import {render} from "@testing-library/react";
import {NavBar} from "./index";

describe('NavBar', () => {
    it('renders correctly', () => {
        expect(render(<NavBar/>).asFragment()).toMatchSnapshot();
    });
});