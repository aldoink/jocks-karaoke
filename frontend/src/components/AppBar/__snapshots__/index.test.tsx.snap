// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar renders correctly 1`] = `
<DocumentFragment>
  .c2 {
  padding: 0.5rem;
  background-color: transparent;
  border: none;
  border-bottom: 1px solid white;
  color: white;
  font-size: 16px;
  caret-color: white;
}

.c2::-webkit-outer-spin-button,
.c2::-webkit-inner-spin-button {
  -webkit-appearance: none;
  margin: 0;
}

.c2[type=number] {
  -moz-appearance: textfield;
}

.c2:focus {
  outline: none;
}

.c2::-webkit-input-placeholder {
  color: white;
  opacity: 1;
}

.c2::-moz-placeholder {
  color: white;
  opacity: 1;
}

.c2:-ms-input-placeholder {
  color: white;
  opacity: 1;
}

.c2::placeholder {
  color: white;
  opacity: 1;
}

.c1 {
  position: relative;
  display: inline-block;
  -webkit-transition: 0.5s;
  transition: 0.5s;
  width: 100%;
  padding: 1rem;
}

.c1 svg {
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  fill: white;
  height: 20px;
  -webkit-align-self: center;
  -ms-flex-item-align: center;
  align-self: center;
  position: absolute;
  right: 1rem;
  bottom: 1.5rem;
}

.c3 {
  box-sizing: border-box;
  width: 100%;
  -webkit-transition: border-bottom-color 0.5s;
  transition: border-bottom-color 0.5s;
}

.c3:focus {
  border-bottom-color: red;
}

.c0 {
  position: -webkit-sticky;
  position: sticky;
  position: -webkit-sticky;
  top: 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

@media screen and (min-width:769px) {
  .c0 {
    margin: 0 2rem;
  }
}

<div
    class="c0"
  >
    <div
      class="c1"
    >
      <input
        class="c2 c3"
        placeholder="Search"
        type="text"
      />
      <svg>
        search-icon.svg
      </svg>
    </div>
    <div
      class=""
      data-testid="hamburger-container"
      id="hamburger-container"
    >
      <button
        class="hamburger hamburger--squeeze "
        data-testid="hamburger-button"
        type="button"
      >
        <span
          class="hamburger-box"
        >
          <span
            class="hamburger-inner"
          />
        </span>
      </button>
    </div>
    <div
      class="menu-drawer"
      data-testid="menu-drawer"
      style="max-height: 0px; visibility: hidden;"
    >
      <div
        class="menu-item"
      >
        <div>
          Login
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
